WARNING:root:Warning: Some classes do not exist in the target. F1 scores for these classes will be cast to zeros.
WARNING:root:Warning: Some classes do not exist in the target. F1 scores for these classes will be cast to zeros.
WARNING:root:Warning: Some classes do not exist in the target. F1 scores for these classes will be cast to zeros.
WARNING:root:Warning: Some classes do not exist in the target. F1 scores for these classes will be cast to zeros.
Epoch 1: Train Loss: 2.8719, Train Vis Loss: 2.8719, Val Loss: 3.1635, Train Acc: 17.33%, Val Accuracy: 9.09%, Train F1 Score: 16.11%, Val F1 Score: 4.17%
For class normal the Train F1: 0.00% and Val F1: 0.00%
For class AVNRT the Train F1: 20.83% and Val F1: 0.00%
For class AVRT the Train F1: 13.33% and Val F1: 16.67%
For class concealed the Train F1: 8.70% and Val F1: 0.00%
For class EAT the Train F1: 21.58% and Val F1: 0.00%
Traceback (most recent call last):
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/runpy.py", line 198, in _run_module_as_main
    return _run_code(code, main_globals, None,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/runpy.py", line 88, in _run_code
    exec(code, run_globals)
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.6.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/__main__.py", line 39, in <module>
    cli.main()
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.6.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 430, in main
    run()
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.6.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 284, in run_file
    runpy.run_path(target, run_name="__main__")
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.6.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 321, in run_path
    return _run_module_code(code, init_globals, run_name,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.6.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 135, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/home/tzikos/.vscode/extensions/ms-python.debugpy-2024.6.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 124, in _run_code
    exec(code, run_globals)
  File "/home/tzikos/Desktop/KULeuvenThesis/main.py", line 131, in <module>
    CVtrain(modelStr=MODEL_STR, learningRate=LEARNING_RATE_BERT, epochs=1000, classWeights=classWeights,
  File "/home/tzikos/Desktop/KULeuvenThesis/trainingFuncs.py", line 229, in CVtrain
    trainCMTable.add_data(*row)
  File "/home/tzikos/anaconda3/envs/KUThesis/lib/python3.12/site-packages/wandb/data_types.py", line 393, in add_data
    raise ValueError(
ValueError: This table expects 6 columns: ['Class', 'Predicted: normal tachy', 'Predicted: AVNRT tachy', 'Predicted: AVRT tachy', 'Predicted: concealed tachy', 'Predicted: EAT tachy'], found 5